sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -33,13 +33,14 @@ export interface EventPositionOptions {
 }
 
 /**
- * Represents options can be set during the creation of a event hub receiver.
- * Defines a position of an @link~EventData in the event hub partition.
- * @class EventPosition
+ * The position of events in an Event Hub partition, typically used in the creation of
+ * an `EventHubProducer`.
```

Comment:
I somehow liked the word "represent" better...

`position of events` doesnt feel right. position is for a single event right?

Reply:
How about
> Represents the position of an event in an Event Hub partition...